### DISCUSSION TITLE
配置IPTV以后首页流量图显示问题

### DISCUSSION THREAD

**Discussion by moonshine00** [istoreos/istoreos#693] [Q&A] [answered]
由于本地网络接口配置了iptv,首页流量图显示是iptv的，没法改成wan口流量图。能否出个设置可以自定义这个流量图的源端口，谢谢。

**Comment by jjm2473** [comment]
难道你的默认网关用的是iptv？理论上只有iptv不应该有默认网关，不然默认会用iptv接口通信

**Comment by moonshine00** [comment]
默认网关是lan.
![截屏2023-03-20 下午1 49 41](https://user-images.githubusercontent.com/34091349/226257803-dcd8ccc0-b1b1-45e2-bbbf-70a741d05d54.png)

**Comment by jjm2473** [comment]
这是怎么看出默认网关是lan的，要在“状态-路由表”里面看好吧

**Comment by moonshine00** [comment]
![截屏2023-03-21 上午10 58 42](https://user-images.githubusercontent.com/34091349/226508197-2d087fd2-e073-4258-997a-8e3b27c87f86.png)

**Comment by moonshine00** [comment]
用mwan3做的分流

**Comment by jjm2473** [comment]
为什么iptv还配置默认网关，一般的流量根本就不会往这里出去。关掉iptv的默认网关配置

**Comment by moonshine00** [comment]
iptv是DHCP+动态获取的

**Comment by jjm2473** [comment]
> iptv是DHCP+动态获取的

总之关掉默认网关，不想跟你普及网络知识。

**Comment by moonshine00** [comment]
请教一下在哪里关

**Comment by jjm2473** [comment, marked as answer]
就在配置iptv接口的地方啊，不是dhcp客户端吗，在旁边有“高级设置”标签页，里面关掉“使用默认网关”

**Reply by 8MiYile** [reply]
不建议这样操作 因为这个操作直接炸飞了IPTV所需的FCC

**Reply by jjm2473** [reply]
FCC是啥？跟默认网关有关？

**Reply by 8MiYile** [reply]
快速换台协议 不过我找到了替代默认网关的方法了

**Reply by jjm2473** [reply]
iptv不是通过组播来获得数据吗，跟默认网关有啥关系？FCC又依赖什么？

**Comment by moonshine00** [comment]
感谢，搞定了，一直没注意这个地方。。。。
